lms/djangoapps/courseware/tabs.py

```diff
@@ -69,7 +69,7 @@ class CourseInfoTab(CourseTab):
     """
     type = 'course_info'
     title = gettext_noop('Home')
-    priority = 20
+    priority = 10
```

Comment:
The reason for setting these to 10 and 11 is that these tabs are paired, and there is not reason or scope to have tabs between they two. One or the other will be active.